sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServices.java

```diff
@@ -150,4 +156,47 @@ void createDataset(
     Table patchTableDescription(TableReference tableReference, @Nullable String tableDescription)
         throws IOException, InterruptedException;
   }
+
+  /** A class for controlling insertAll submission rate. */
```

Comment:
JavaDoc describing bit more about what it is controlling?

Reply:
added some comments.